models/sessions/scratch/sessions_scratch.yml

```diff
@@ -41,7 +41,8 @@ models:
       - name: engaged_time_in_s
         description: '{{ doc("col_session_engaged_time_in_s") }}'
         tests:
-          - not_null
+          - not_null:
```

Comment:
Let's change the base to the release branch!

Reply:
ah ok will do 
Is this the release branch?
Release/snowplow-unified/0.5.0

Reply:
yes, I think we can release this together with the Spark updates.